advanced_functionality/unity_ml_agents/src/train.yaml

```diff
@@ -0,0 +1,26 @@
+behaviors:
```

Comment:
ml-agent YAML file is version specific. It'll be good to have some notation about what ml-agent version the notebook is using

Reply:
added some notations